tcp_protocol.json

```diff
@@ -832,7 +832,18 @@
                         "lower_limit": -1
                     }
                 ]
+            },
+            {
+                "name": "thickness_gauge",
```

Comment:
Should this rather be `set_thickness_gauge_sound_veloctiy`?

Will this be more variables set later?